File: Examples_VI_CtCI_Java/src/Example1.java
public class Example1 {
	// 0 (N) runtime.
	
	static void foo(int[] array) {
		int sum = 0;
		int product = 1;
		
		for (int i = 0; i < array.length; i++) {
			sum += array[i];
		}
		for (int i = 0; i < array.length; i++) {
			product *= array [i];
		}
		System.out.println(sum + ", " + product);
	}
	
	public static void main(String[] args) {
		int[] array = {1, 2, 3, 4, 5};
		
		foo(array);
	}
}

File: Examples_VI_CtCI_Python/Example1.py
def foo(array):
    sum = 0
    product = 1
    i = 0
    while (i < len(array)):
        sum += array[i]
        i += 1
    i = 0
    while (i < len(array)):
        product *= array[i]
        i += 1
    print(str(sum) + ", " + str(product))

array = [1,2,3,4,5]
foo(array)